### PR TITLE
add more instruction for the redis deployment

### PR DESCRIPTION
**Pull request by lwaekfjlk**
Closes # 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed
- [x] Branch name follows `type/descript` (e.g. `feature/add-llm-agents`)
- [x] Ready for code review

### CODE REVIEW

File: docs/all_the_issues.md [hunk 1]
```diff
@@ -26,4 +26,18 @@ Then you can start your database with:
...
+export REDIS_OM_URL="redis://:QzmCUD3C3RdsR@localhost:6379"
+```
+
+instead of adding variables in the python code
+
+```sh
+os.environ['REDIS_OM_URL'] = "redis://:QzmCUD3C3RdsR@localhost:6379"
+```
+
+since we need to set a global environment variable that works for `redis_om`.
```
Comment (ProKil) [inline docs/all_the_issues.md:43]: This is useful, maybe it could a new FAQ entry? What is the symptom of doing this wrong?

Comment (lwaekfjlk) [reply docs/all_the_issues.md:43]: if we set this, I came across authorization error.

Comment (ProKil) [reply docs/all_the_issues.md:43]: Cool! Would you mind moving your answer under a new question like "Why do I encounter authorization error '<input the concrete error message>' after setting `REDIS_OM_URL` in Python `os.environ`"?

Comment (lwaekfjlk) [reply docs/all_the_issues.md:43]: yes, will do later.

Comment (lwaekfjlk) [reply docs/all_the_issues.md]: UPDATED!

**Review by ProKil — APPROVED** [review]

**Pull request merged**